packages/mdx-embed/src/components/youtube/youtube.tsx

```diff
@@ -30,6 +33,11 @@ export const YouTube: FunctionComponent<IYouTubeProps> = ({
 
   const startTime = tH + tM + s;
 
+  const baseUrl = 'https://www.youtube.com/embed/';
```

Comment:
A change has been merged that conditionally decides if the "no cookie" url should be used so you might have conflicts on this.